lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -274,13 +290,14 @@ mapping:
 
       conda_prefix:
         type: str
-        default: <tool_dependency_dir>/_conda
```

Comment:
I was thinking, should this instead be set to `default: _conda`, to be interpreted w.r.t. to `self.tool_dependency_dir` ?

Same goes for `tool_dependency_cache_dir` and `involucro_path`

Reply:
Yes. And I believe there are more like that. I haven't touched them for the sake of keeping this PR small enough. I'd rather make such additional changes in a follow-up. I'll make this edit, but I'd rather wait with similar edits until this one gets merged.

Reply:
Correction. I do prefer to leave this until these are merged. `conda_prefix` is not used in `config/__Init__.py`, but it is used (and resolved) in `tool_util/deps/resolvers/conda.py` and `tool_util/deps/conda_util.py`. If/when we set this default, we should also revise how that value is resolved in those two files. I believe this should be left for a different PR. I'm adding it to my TODO list.

Reply:
Marking as resolved for now, to cleanup this PR. Will make this change in a follow-up. (if you'd rather leave this as not resolved for now, that's fine - please reopen.)